src/Hazelcast.Net.Tests/Networking/SslLayerTests.cs

```diff
@@ -58,7 +58,9 @@ public void ValidateCertificate()
             var loggerFactory = LoggerFactory.Create(builder => builder.AddStringBuilder(text));
 
             var sender = new object();
+#pragma warning disable SYSLIB0026
```

Comment:
add a comment: "X509Certificate empty constructor is obsolete BUT here we DO want to use it to create an empty cert"

Reply:
Fixed at https://github.com/hazelcast/hazelcast-csharp-client/pull/682/commits/146cc487e6723800e0609781507b11b70e1c1c41

Reply:
ok